app/javascript/article-form/articleForm.jsx

```diff
@@ -284,6 +285,7 @@ export class ArticleForm extends Component {
     if (!revert && navigator.userAgent !== 'DEV-Native-ios') return;
 
     this.setState({
+      formKey: new Date().toISOString(),
```

Comment:
suggestion (not blocking) - I think it might be worth adding a comment here to explain why we set this key